### ISSUE TITLE
3 in input data?

### ISSUE THREAD

**Issue by vasmue** [LegoCreation/CNN_eddy_detection#1]
https://github.com/LegoCreation/CNN_eddy_detection/blob/4b83f105650bc554fc46ea508a07ed71dffd03d7/scripts/cnn_training.py#L68

Max noticed that sometimes there are 3's in the seg mask. Seems like you also had this problem. Any idea why?

**Comment by LegoCreation** [comment]
I didn't have those problems when I tried those. I create those lines just to be extra sure.

**Comment by vasmue** [comment]
https://github.com/LegoCreation/CNN_eddy_detection/blob/4b83f105650bc554fc46ea508a07ed71dffd03d7/scripts/generate_segmentation_mask.py#L71C57-L71C57

looks like it comes from here. If both masks are 1, then the result will be 3

**Comment by LegoCreation** [comment]
3 is possible if you have both anticylonic and cyclonic eddy pixel at the same spot.

**Comment by vasmue** [comment]
Yes, exactly this. But it should theoretically never happen (eddy can't be both), so I am a bit confused why py-eddy-tracker does it sometimes.
